src/amplify-migration/githubUtils.ts

```diff
@@ -61,3 +61,12 @@ export async function isRepoEmpty(repoName: string): Promise<boolean> {
     throw e;
   }
 }
+
+export function isRepoMigrated(repoPath: string): boolean {
+  // read the config.yml file
+  const configFile = fs.readFileSync(`${repoPath}/_config.yml`, "utf-8");
+  if (configFile.includes(".amplifyapp.com")) {
+    return true;
+  }
+  return false;
```

Comment:
nit: can probably return `configFile.includes(".amplifyapp.com")`

also, there were cases previously where the config file didn't update but our db did (ie, our db is the source of truth here). given that this is a script, maybe we wanna log this to the file and ask them to check manually

Reply:
@seaerchin The caller of this function already logs if the repo is already migrated! 
also, this does not work if "where the config file didn't update but our db did", because the only moment where this error will come to light is when eng runs the sql commands to which then we will get conflict errors. 
I think this is ok as 
1. errors are not silent 
2. Side effects are clearer now when looking at the logs, eng should be able to remediate this edge case. 


Abit of context here is that this change was introduced because there exists human error when ops gives us the list (which should be on us to encounter for)

Reply:
yeap, for clarity's sake, what i meant is that we should be logging here (or in the caller) when the config file does **not** include the `amplifyapp` post migration. 

this would give more salient feedback faster also

Reply:
> yeap, for clarity's sake, what i meant is that we should be logging to the file (or in the caller) when the config file does not include the amplifyapp post migration.

hey @seaerchin I hear you on this, but I realised that this is quite easily solvable! Problem stems when the site migrated is an older site that does not have the `staging` + `prod` key. Rather than logging + making eng add in the two lines, opting for the quick win [here](https://github.com/isomerpages/isomer-conversion-scripts/pull/70/commits/5c472d874034528a212531f53ec5faecbc1ea40d) of just appending the key value pair directly onto the `config.yml` file to reduce the mental overhead required.